cms/djangoapps/contentstore/views/course.py

```diff
@@ -643,7 +643,7 @@ def _accessible_libraries_iter(user, org=None):
         libraries = [] if org == '' else modulestore().get_libraries(org=org)
     else:
         libraries = modulestore().get_library_summaries()
-    # No need to worry about ErrorBlocks - split's get_libraries() never returns them.
+
```

Comment:
I think this might be because he's using it to mix two different key types (which I also think is worth examining). That could be done with a `defaultdict`, but it'd need a lambda to switch by type or something, which is getting into 😦 territory.

Reply:
@ormsbee did you mean to put this on https://github.com/openedx/edx-platform/pull/34523#discussion_r1579833238 ?

Reply:
... yeah. Weird. Thank you.

Reply:
Okay thanks, I'll scrub my PR for things like this next time.